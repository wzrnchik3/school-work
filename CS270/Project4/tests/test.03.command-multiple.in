echo does this work?
echo what about this?
