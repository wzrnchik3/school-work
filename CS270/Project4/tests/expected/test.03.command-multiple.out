does this work?
what about this?
